src/Components/SmartComponents/Reports/Common/firstPagePDF.js

```diff
@@ -4,55 +4,58 @@ import { Paragraph } from '@redhat-cloud-services/frontend-components-pdf-genera
 import { Text, View } from '@react-pdf/renderer';
 import messages from '../../../../Messages';
 import tablePage from './tablePage';
+import { PUBLIC_DATE_OPTIONS } from '../../../../Helpers/constants';
 
 const firstPagePDF = ({ cves, meta, filters, intl, isReportDynamic, reportData, user }) => {
     // if data isn't converted to object convert it
     const filtersBuilder = (isReportDynamic) ? filters : filters.reduce((obj, item) => {
         obj[item.key] = {
             filter: item.category,
-            values: item.chips.map(({ name }) => name).join(', ')
+            values: item.key === 'publish_date'
+                ? PUBLIC_DATE_OPTIONS.find(val => val.value === item.chips[0].value).timespan_label
+                : item.chips.map(({ name }) => name).join(', ')
         };
         return obj;
     }, {});
 
     const generateFiltersParagraph = () =>
-        // if there are no filters don't dispay this paragraph
-        // if there is at least one start with "This report includes CVEs with "
-        // if has CVSS          += "a CVSS base score of <min>-<max>"
-        // if has publish_date  += "publish date" -> custom intl -> "withing 7 days ... 1 year OR more than 1 year ago"
-        // if has severity      += "severity of <list>"
-        // if has business risk += "business risk of <list>"
-        // if has status        += "status of <list>"
-        // if has security rule += "<option>" (has security rule)
-
-        Object.keys(filtersBuilder).length === 0 ? undefined : (<Paragraph>
-            {intl.formatMessage(messages.CVEsPDFreportHeader)}
-            {filtersBuilder.hasOwnProperty('cvss_filter') && intl.formatMessage(messages.CVEsPDFreportFiltersCvssScore, {
-                score: filtersBuilder.cvss_filter.values,
-                b: (...chunks) => <Text style={{ fontWeight: 700 }}>{chunks}</Text>
-            })}
-            {filtersBuilder.hasOwnProperty('publish_date') && intl.formatMessage(messages.CVEsPDFreportFiltersPublishDate, {
-                date: filtersBuilder.publish_date.values,
-                b: (...chunks) => <Text style={{ fontWeight: 700 }}>{chunks}</Text>
-            })}
+        <Paragraph>
+            {filtersBuilder.hasOwnProperty('filter')
+                ? intl.formatMessage(messages.customReportSearchTermCvssAndPublishDate, {
+                    search_term: filtersBuilder.filter.values,
+                    score: (filtersBuilder.cvss_filter && filtersBuilder.cvss_filter.values.toLowerCase()) || '0.0 - 10.0',
```

Comment:
it would be better to use the default CVSS score. `DEFAULT_FILTER_DATA`

Reply:
`DEFAULT_FILTER_DATA.cvss_filter.min.toFixed(1) + ' - ' + DEFAULT_FILTER_DATA.cvss_filter.max.toFixed(1);`
is better but way too long :frowning_face:

Reply:
You can use different ways to display it


 `${DEFAULT_FILTER_DATA.cvss_filter.min.toFixed(1) - ${DEFAULT_FILTER_DATA.cvss_filter.max.toFixed(1)};`
```
 let { cvss_filter: { min, max } } = DEFAULT_FILTER_DATA;
`${min.toFixed(1)} - ${max.toFixed(1)}|`
```

or even 

```
let { cvss_filter } = DEFAULT_FILTER_DATA
Object.keys(cvss_filter).map(k => cvss_filter[k].toFixed(1)).join(' - ')

```